### PR TITLE
Add stink lines

### PR DESCRIPTION
**Pull request by Sovexe**
## About the PR 
https://streamable.com/si0sf9

## Why's this needed? 
* fixes bug where you could never lose the stink effect
* for fun

## Changelog

```changelog
(u)Sovexe
(+)Added stink lines for smelly RP players
```

### CODE REVIEW

File: code/datums/sims.dm [hunk 1]
```diff
@@ -279,7 +281,7 @@
 				showOwner("<span class='alert'>You are [pick("disgusted", "revolted", "repelled", "sickened", "nauseated")] by your own [pick("smell", "odor", "body odor", "scent", "fragrance", "bouquet", "savour", "tang", "whiff")]!</span>")
 			*/
 			#ifdef CREATE_PATHOGENS //PATHOLOGY_REMOVAL
-			if (value < 5 && prob(1) && prob(25))
+			if (value < 5 && prob(1) && prob(25))//
```
Comment (Sovexe) [inline code/datums/sims.dm:284]:
```suggestion
			if (value < 5 && prob(1) && prob(25))
```

File: code/obj/effects/particles.dm [hunk 2]
```diff
@@ -135,3 +135,16 @@ particles/spark
 	position = generator("circle", 0, 64, NORMAL_RAND)
 	friction = generator("num", 0, 0, NORMAL_RAND)
 	drift = generator("box", list(-0.1,-0.1,0), list(0.1,0.1,0), UNIFORM_RAND)
+
+/particles/stink_lines
+	icon = 'icons/effects/particles.dmi'
+	icon_state = list("line")
+	color = generator("color", "#808000", "#808000", NORMAL_RAND)
```
Comment (Sovexe) [inline code/obj/effects/particles.dm:142]:
```suggestion
	color = generator("color", "#808000", "#806900", NORMAL_RAND)
```

**Review by Sovexe — COMMENTED** [review]
Fix

**Comment by Connie-Dun** [comment]
I think it's funny but it's honestly not a thing I'd like being a thing
Sometimes it's just annoying to need to go do Stuff for motives while you're in the middle of a heavy rp scene
I like how as-is the low-motives consequences aren't really visual, just stuff in the chatbox
I get from a gameplay perspective why the motives are designed to deplete within an hour and a half round
but I'm also not a fan of how that feels from a RP perspective, and a visual change if you don't go maintain motives just exacerbates it
no matter how much you eat/drink/clean early on, you WILL become ravenously hungry/thirsty/smelly
and it all being tucked in the chatbox makes it easier to ignore when need be

**Comment by Sovexe** [comment]
Going to try it out for now and evaluate. Maybe also adjust or expand upon the motive. As it is currently going from 0 to 100% hygiene is pretty trivial between sinks, showers, pools, beakers, and even just puddles of water.

**Comment by Yellow-Mushroom** [comment]
If im being honest, while I do like that this encourages showering and what not, it does appear a bit weird. Perhaps a bit too opaque while also being small and hard to read?

**Pull request merged**